### ISSUE TITLE
Status List is not being updated properly in Status Update

### ISSUE THREAD

**Issue by WorkMAE** [WorkMAE/CEM-Integration#44]
Status History list is being updated but the status list itself is not being updated.

**Comment by WorkMAE** [comment]
This is now resolved.  The issue occurred when I was looking at the last date for an item in the status history grid. If there was no previous item in the status history grid, it returned an empty array instead of the current status for that service.  This is now resolved and in turn it resolves the issue with the "Multiple" items not showing up properly as well.

Please verify this works properly and then close.

**Comment by WorkMAE** [comment]
Fixed more issues involving the status update grid.  

- fixed an issue where the saved project status(if single item) in the status list was always the status being updated regardless of date. It now updates properly to the latest status based on date

- same as above for the Current Status header after updating---it now shows the latest project status for the items based on date instead of based on last updated status

- fixed an issue where items that were deleted or updated tried to update and delete on every subsequent status change(if you had multiple ones you were doing).  the deletion and change arrays are now set to empty after response is received back from the server.

- fixed an issue where a single project status would not update another service to the same status since it was looking to see if the statuses matched.  I removed this check as it is no longer relevant since we are updating multiple statuses now instead of a single status.

I believe this should resolve all outstanding issues with the status update functionality, I will continue testing it to ensure this is working as intended.
